### PR TITLE
Stoplight elements openapi

### PR DESCRIPTION
**Pull request by aedify-swi**
- Adds support for stoplight elements openapi schema visualisation mirroring swagger options.
- Adds respective docs (and fixes a minor typo).

### CODE REVIEW

File: starlite/openapi/controller.py [hunk 1]
```diff
@@ -104,6 +105,37 @@ def swagger_ui(self, request: Request) -> str:
...
+            <meta charset="utf-8"/>
+            <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+            <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements@{self.stoplight_elements_version}/styles.min.css">
+            <script src="https://unpkg.com/@stoplight/elements@{self.stoplight_elements_version}/web-components.min.js" crossorigin></script>
+          </head>
+        """
+        body = f"""
+          <body>
+            <elements-api
+            apiDescriptionUrl="{self.path}/openapi.json"
```
Comment (Goldziher) [inline starlite/openapi/controller.py:125]: indentation

**Review by Goldziher — APPROVED** [review]
Looks good, please update the readme.md as well.

**Comment by Goldziher** [comment]
@all-contributors add @aedify-swi code

File: starlite/openapi/controller.py (hunk 1 above)
Comment (aedify-swi) [reply starlite/openapi/controller.py]: fixed

**Review by Goldziher — APPROVED** [review]

**Pull request merged**